keps/sig-network/3705-cloud-node-ips/README.md

```diff
@@ -600,53 +290,30 @@ to start after downgrade.
 ###### How can this feature be enabled / disabled in a live cluster?
 
 - [X] Feature gate (also fill in values in `kep.yaml`)
-  - Feature gate name: CloudNodeIPs
+  - Feature gate name: CloudDualStackNodeIPs
   - Components depending on the feature gate:
     - kubelet
     - cloud-controller-manager
 
 ###### Does enabling the feature change any default behavior?
 
-Kubelet will begin setting two annotations on each node rather than
-one. However, cloud providers will still prefer the old annotation and
-will only read the new annotation if the old one isn't there.
-
-So assuming no bugs, the only visible change after (just) enabling the
-feature is the duplicate annotation.
+No
 
 ###### Can the feature be disabled once it has been enabled (i.e. can we roll back the enablement)?
 
 Yes, as long as you also roll back the kubelet configuration to no
 longer use the new feature.
```

Comment:
nit: can you just add "... the new feature *earlier or together with the feature disblement* ?